src/apps/editor/components/matrix/form-renderer.tsx

```diff
@@ -31,6 +31,8 @@ export const FormRenderer: React.FC<Props> = ({
   onFieldUpdate,
   onRemoveField,
 }) => {
+  const [collapsed, setCollapsed] = useState<boolean>(false);
```

Comment:
Typing `useState` is redundant as the type is inferred from the `initialValue` definition. I would only use it in complex cases such as objects, maps, sets, arrays...

```tsx
export const Component: React.FC = () => {
    // ...
    // const [state, dispatcher] = useState([]); // TS cannot infer the type of the array 👎🏼
    const [state, dispatcher] = useState<string[]>([]);
    // ...
}
```